com.kolor.AutopanoPro.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Kolor -->
+<component type="desktop">
+    <id>com.kolor.AutopanoPro</id>
+    <metadata_license>LicenseRef-proprietary</metadata_license>
```

Comment:
this is not acceptable. THIS file should be under a license that allow its redistribution. `CC0-1.0` is preferred.

Reply:
yeah I messed those two lines up sorry about that